priv/templates/_action_dialog_new_rsc_tab.tpl

```diff
@@ -36,6 +48,16 @@
                 {_ Published _}
             </label>
         </div>
+    {% elseif predicate == 'hasextra_rsc' %}
```

Comment:
AFAICT this branch is the same as the one for `hasextra_faq`, can you merge them by adding an `or` in the previous condition?